doc/authorization.md

```diff
@@ -0,0 +1,165 @@
+---
+relatedlinks: https://discourse.ubuntu.com/t/identity-and-access-management-for-lxd/41516
+---
+
+(authorization)=
+# Remote API Authorization
+
+When LXD is {ref}`exposed over the network <server-expose>` it is possible to restrict API access via two mechanisms:
+
+- {ref}`restricted-tls-certs`
+- {ref}`fine-grained-authorization`
+
+(restricted-tls-certs)=
+## Restricted TLS certificates
+
+It's possible to restrict a {ref}`TLS client <authentication-trusted-clients>` to one or multiple projects.
+In this case, the client will also be prevented from performing global configuration changes or altering the configuration (limits, restrictions) of the projects it's allowed access to.
+
+To restrict access, use [`lxc config trust edit <fingerprint>`](lxc_config_trust_edit.md).
+Set the `restricted` key to `true` and specify a list of projects to restrict the client to.
+If the list of projects is empty, the client will not be allowed access to any of them.
+
+(fine-grained-authorization)=
+## Fine-grained authorization
+
+It is possible to restrict {ref}`OIDC clients <authentication-openid>` to granular actions on specific LXD resources.
+For example, one could restrict a user to be able to view, but not edit, a single instance.
+
+There are four key concepts that LXD uses to manage these fine-grained permissions:
+
+- __Entitlements__: An entitlement encapsulates an action that can be taken against a LXD API resource type.
+   Some entitlements might apply to many resource types, whereas other entitlements can only apply to a single resource type.
+   For example, the entitlement `can_view` is available for all resource types, but the entitlement `can_exec` is only available for LXD resources of type `instance`.
+- __Permissions__: A permission is the application of an entitlement to a particular LXD resource.
+   For example, given the entitlement `can_exec` which is only defined for instances, a permission is the combination of `can_exec` and a single instance, as uniquely defined by its API URL i.e. `/1.0/instances/c1?project=foo`.
+- __Identities (users)__: An identity is any authenticated party that makes requests to LXD, including TLS clients.
+   When an OIDC client adds a LXD server as a remote, the OIDC client is saved in LXD as an identity.
+   Permissions cannot be assigned to identities directly.
+- __Groups__: A group is a collection of one or more identities.
+   Identities can belong to one or more groups.
+   Permissions can be assigned to groups.
+   TLS clients cannot currently be assigned to groups.
```

Comment:
So that means that you cannot assign a permission to a TLS client?
(Just to make sure I understand it correctly from what is written.)

Reply:
No. TLS clients are still using the existing TLS authorization driver until we have fixed any issues and validated the performance of fine-grained auth.